RELEASE_NOTES.md

```diff
@@ -1,6 +1,7 @@
 
 # Next Release
 
+- [#198](https://github.com/IAMconsortium/pyam/pull/198) Update stack plot functionality and add aggregation tutorial.
```

Comment:
Can you also add a note about the new `copy()` function? I think people will find it quite useful!